vault/logical_system_paths.go

```diff
@@ -371,6 +372,7 @@ func (b *SystemBackend) rekeyPaths() []*framework.Path {
 				},
 			},
 
+			// TODO not sure what to do for this as there are no callbacks
```

Comment:
_think_ it's here
https://github.com/hashicorp/vault/blob/4b8bd15e28ad3194159e7601fe76a994688988d4/http/sys_rekey.go

(@AnPucel helped me last time one was missing iirc)

Reply:
Yeah. Some of the `sys` calls don't have callbacks they get routed directly. If you code search the path it will show you where it gets handled, which I believe is [here](https://github.com/hashicorp/vault/blob/4758cc8f8646f93787df1f6b9755d3947ee6321c/http/sys_rekey.go#L16)